src/components/20-molecules/policy-features/index.scss

```diff
@@ -0,0 +1,49 @@
+.m-policy-features {
+  padding: 24px 15px;
+
+  @include breakpoint($mediaquery-md-up) {
+    padding: 48px 90px;
+  }
+
+  &__title {
+    text-align: center;
+    margin: 0;
+
+    @include typo-title(large-1, secondary);
+
+    @include breakpoint($mediaquery-md) {
+      @include typo-title(huge, secondary);
+    }
+
+    @include breakpoint($mediaquery-lg-up) {
+      @include typo-title(huge-2, secondary);
+    }
+  }
+
+  &__slot {
+    display: flex;
```

Comment:
Is that according to the styleguide? Because there are situations, where we get 2 rows because of the screen size, an the first row contains 3 items and the second only 1.

Reply:
This situation exists in V1 too. I will ask a designer... good point.

Reply:
Designer said its ok.